_provisioning/configuration/etc/ramiel/ws-scrcpy/settings_local_sample.json

```
@@ -1,6 +1,16 @@
 {
-  "serverPort": 28500,
+  "AES_0": "aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa",
+  "AES_1": "bbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbb",
+  "AES_2": "cccccccccccccccccccccccccccccccc",
+  "AES_3": "dddddddddddddddddddddddddddddddd",
+  "AES_4": "eeeeeeeeeeeeeeeeeeeeeeeeeeeeeeee",
+  "AES_5": "ffffffffffffffffffffffffffffffff",
+  "AES_6": "gggggggggggggggggggggggggggggggg",
+  "AES_7": "hhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhh",
+  "AES_8": "iiiiiiiiiiiiiiiiiiiiiiiiiiiiiiii",
+  "AES_9": "jjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjj",
```

@metocks // kaji 처럼 서버쪽에 AES_0 만 있는게 좋을것 같습니다. 서버가 털려도 0번 키만 교체하면 되도록 하기 위함입니다.
https://github.com/HardBoiledSmith/kaji/blob/28c44cb74e4d73a5c301fce0caad821e7a1e8796/lambda/connect_call/settings_local_sample.py#L1
```suggestion
  "AES_0": "aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa",
```